### PR TITLE
Issue 113 doc pages

### PR DESCRIPTION
**Pull request by amol-anand**
Added doc support to leadspace for document pages.

Fix: #113

Test URLs:
Before: https://main--merative2--hlxsites.hlx.page/drafts/amol/pdf-viewer-new
After: https://issue-113-doc-pages--merative2--hlxsites.hlx.page/drafts/amol/pdf-viewer-new

### CODE REVIEW

**Comment by amol-anand** [comment]
Consistently getting 100 LHS on both mobile and desktop directly here: https://pagespeed.web.dev/report?utm_source=psi&utm_medium=redirect&url=https%3A%2F%2Fissue-113-doc-pages--merative2--hlxsites.hlx.page%2Fdrafts%2Famol%2Fpdf-viewer-new

File: blocks/leadspace/leadspace.css [hunk 1]
```diff
@@ -16,22 +16,69 @@ main div.leadspace-wrapper {
...
 .leadspace > div > div {
     position: relative;
 }
 
-.leadspace > div > div:nth-child(1) {
+.leadspace:not(.document) > div > div:nth-child(1) {
     margin: auto;
 }
 
+.leadspace.document .document-tag-container {
```
Comment (arumsey) [inline blocks/leadspace/leadspace.css:41]: @amol-anand This document variant you've added isn't included as part of the Leadpapce design in Figma. Have you confirmed this variant should be included as part of the LEadspace block vs a different block that is unique to the document viewer page?

**Comment by arumsey** [comment]
@amol-anand Could you provide a link to what the authored Word document would look like?

File: blocks/leadspace/leadspace.css [hunk 2]
```diff
@@ -16,22 +16,69 @@ main div.leadspace-wrapper {
...
+}
+
+.leadspace.document .readtime {
+    font-family: var(--sans-serif-font-medium);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 14px;
+    padding: 4px 0;
+    margin-right: var(--spacer-element-07);
+    line-height: 120%;
```
Comment (arumsey) [inline blocks/leadspace/leadspace.css:54]: Could use `--line-height-120`

File: blocks/leadspace/leadspace.css [hunk 3]
```diff
@@ -16,22 +16,69 @@ main div.leadspace-wrapper {
...
+    display: flex;
+    flex-flow: row wrap;
+    row-gap: var(--spacer-element-03);
+}
+
+.leadspace.document .readtime {
+    font-family: var(--sans-serif-font-medium);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 14px;
```
Comment (arumsey) [inline blocks/leadspace/leadspace.css:51]: `--font-size-14`

File: blocks/leadspace/leadspace.css [hunk 4]
```diff
@@ -16,22 +16,69 @@ main div.leadspace-wrapper {
...
+    padding: 4px 0;
+    margin-right: var(--spacer-element-07);
+    line-height: 120%;
+    letter-spacing: 0.2em;
```
Comment (arumsey) [inline blocks/leadspace/leadspace.css:55]: --letter-spacing-02-em

File: blocks/leadspace/leadspace.css [hunk 5]
```diff
@@ -16,22 +16,69 @@ main div.leadspace-wrapper {
...
+
+.leadspace.document .document-tag {
+    padding: 4px 16px;
+    background: var(--neutral-sand);
+    border-radius: 100px;
+    margin-right: var(--spacer-element-03);
+}
+
+.leadspace.document .download-link-container {
+    padding-top: 16px;
```
Comment (arumsey) [inline blocks/leadspace/leadspace.css:68]: `--spacer-layout-01`

File: blocks/leadspace/leadspace.css [hunk 6]
```diff
@@ -16,22 +16,69 @@ main div.leadspace-wrapper {
...
+}
+
+.leadspace.document a.download-link {
+    font-family: var(--sans-serif-font-regular);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 120%;
+    color: #90F;
```
Comment (arumsey) [inline blocks/leadspace/leadspace.css:77]: Is there a var for thius color?

**Review by arumsey — COMMENTED** [review]
There are a bunch of styles that could be rpelaced by vars.

File: blocks/leadspace/leadspace.css [hunk 7]
```diff
@@ -16,22 +16,69 @@ main div.leadspace-wrapper {
...
+    padding-right: 24px;
+    background: url('../../icons/arrow-down-download-purple.svg') no-repeat right center;
```
[the remark this answers is not in the document]
Comment (arumsey) [reply blocks/leadspace/leadspace.css:79]: Does the download arrow have a different design?

File: blocks/leadspace/leadspace.css (hunk 1 above)
Comment (amol-anand) [reply blocks/leadspace/leadspace.css:41]: good point. it looks and functions exactly like the Leadspace which is why it makes sense as compared to a separate Leadspace just for document pages. I'll check with the Design team anyways.

**Comment by amol-anand** [comment]
> Could you provide a link to what the authored Word document would look like?
It's in the PR description :) 
https://issue-113-doc-pages--merative2--hlxsites.hlx.page/drafts/amol/pdf-viewer-new.md

File: blocks/leadspace/leadspace.css (hunk 7 above)
[the remark this answers is not in the document]
Comment (amol-anand) [reply blocks/leadspace/leadspace.css:79]: Yes, arrow-purple did look a little bolder to me which is why i picked this up from the figma file to use. I'm sure there's some redundancy in general which we can clean up after the fact as well.

File: blocks/leadspace/leadspace.css (hunk 2 above)
Comment (amol-anand) [reply blocks/leadspace/leadspace.css]: good catch - fixed

File: blocks/leadspace/leadspace.css (hunk 3 above)
Comment (amol-anand) [reply blocks/leadspace/leadspace.css]: good catch - fixed

File: blocks/leadspace/leadspace.css (hunk 4 above)
Comment (amol-anand) [reply blocks/leadspace/leadspace.css]: good catch - fixed

File: blocks/leadspace/leadspace.css (hunk 5 above)
Comment (amol-anand) [reply blocks/leadspace/leadspace.css]: good catch - fixed

File: blocks/leadspace/leadspace.css (hunk 6 above)
Comment (amol-anand) [reply blocks/leadspace/leadspace.css]: yes, fixed

**Review by arumsey — APPROVED** [review]

**Pull request merged**